integration-testing/rnode_testing/rnode.py

```diff
@@ -470,7 +480,7 @@ def create_peer_nodes(
 
 
 @contextmanager
-def bootstrap_node(docker, docker_network, timeout, validators_data, *, container_name=None, cli_options=None, mount_dir=None):
+def bootstrap_node(docker: "DockerClient", docker_network: str, timeout: int, validators_data: "ValidatorsData", *, container_name: str = None, cli_options=None, mount_dir: str = None) -> Generator[Node, None, None]:
```

Comment:
Why `Generator[Node, None, None]` and not `Generator[Node]`?

Reply:
The `Generator` expects 3 type arguments. If you put only one argument, mypy would report as an error.